Dockerfile

```diff
@@ -42,13 +44,19 @@ COPY meson-cross-i386 /build/
 #
 ARG PLATFORM=
 
+# Set to "y" to enable or something else to disable link time optimizations.
+ARG BUILD_WITH_LTO=y
+
 # Leave empty or comment out to skip LLVM build
 ARG BUILD_WITH_LLVM=
 
 # Set to the desired number of parallel build jobs; this should losely
 # correspond to the number of CPU cores.
 ARG BUILD_JOBS=8
 
+# Set to Wine's installation directory on your machine, e.g. $HOME/.local.
+ARG PREFIX=
```

Comment:
Minor: Since this isn't optional you could put it before the optional ones under `ARG PLATFORM=`

Reply:
Done, thanks.